package.json

```diff
@@ -18,15 +18,16 @@
   "dependencies": {
     "automat": "1.2.0",
     "chai": "^3.5.0",
-    "extend-me": "2.2.4",
-    "filter-tree": "0.3.39",
+    "extend-me": "^2.3",
+    "filter-tree": "0.3.40",
```

Comment:
Shouldn't we able to remove this. Filter-tree is now the first of many implementations that can concur with the grid. However its not mandated.

I spoke to ricardo about mimicing @mrClapham's style of tutorial repos like so https://github.com/openfin/openfin-tutorial-hello-world